server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -371,10 +371,10 @@
             "patch": {
               "patchStrategy": "replace",
               "mutatedRef": [
-                ["settings", "metadata", "name"],
-                ["settings", "metadata", "namespace"],
-                ["settings", "metadata", "kind"],
-                ["settings", "metadata", "apiVersion"]
+                ["settings", "regarding", "name"],
+                ["settings", "regarding", "namespace"],
+                ["settings", "regarding", "kind"],
+                ["settings", "regarding", "apiVersion"]
```

Comment:
eh? Really?

Reply:
@RipulHandoo ?

Reply:
Yes @leecalcote, regarding contains the **_object_** this Event is about. 
There is one more field `related` but that is optional (means it is used when regarding object triggers the creation and deletion of related object).

Example: 
```
apiVersion: v1
kind: Event
metadata:
  name: deployment-event
  namespace: default
  labels:
    type: deployment-related
    component: deployment-controller
data:
  message: "Deployment 'example-deployment' scaled successfully"
  deploymentName: example-deployment
  eventType: Normal
regarding:
  apiVersion: apps/v1
  kind: Deployment
  name: example-deployment
  namespace: default
  ```